core/controllers/admin.py

```diff
@@ -293,17 +293,16 @@ def _create_dummy_question(
 
         state.update_recorded_voiceovers(recorded_voiceovers)
         state.update_written_translations(written_translations)
-        solution_dict = (
-            state_domain.Solution(
-                'TextInput', False, 'Solution', state_domain.SubtitledHtml(
-                    'solution', '<p>This is a solution.</p>')).to_dict())
+        solution = state_domain.Solution(
```

Comment:
Instead of using the class directly, can you use `state_domain.Solution.from_dict`? Since, if the Solution class is changed, from_dict would be easier to modify in the future.

Ditto elsewhere